_docs/javascript-api.md

```diff
@@ -152,31 +152,34 @@ rpc.exports = {
 
 const co = require('co');
 const frida = require('frida');
-const load = require('frida-load');
+const fs = require('fs');
+const path = require('path');
+const readFileAsync = bluebird.promisify(fs.readFile);
```

Comment:
Missing require for `bluebird`. (Requires should be in alphabetic order, so should be added a bit above.)